app/views/advanced/_guided_search_fields.html.erb

```diff
@@ -1,6 +1,7 @@
 
 <div class="advanced-search-field">
   <div class="search-field">
+    <label class="sr-only">Options for advanced search</label>
```

Comment:
@christinach When the label element is detached from the input element, the `for` attribute should be used (the value should be the `id` of the form element that is being labeled. In this case, `<label for ="f1" class="sr-only">Options for advanced search</label>` since the `select` element's `id="f1"`

Reply:
Thank you @axamei; will update the file